lib/encoders/date_time.ex

```diff
@@ -0,0 +1,5 @@
+defimpl Poison.Encoder, for: Ecto.DateTime do
+  def encode(date_time, options) do
+    date_time |> Ecto.DateTime.to_string |> Poison.encode!(options)
```

Comment:
Thank you! I have actually changed this on Ecto master, we should call `Ecto.DateTime.to_iso8601`. Can you please update the mix.exs to point to latest Ecto? Also, we don't need to call Poison.encode! and we need tests too! :)